### DISCUSSION TITLE
Nemo - modern browsing

### DISCUSSION THREAD

**Discussion by Jopp-gh** [linuxmint/discussions#430] [Ideas]
With "modern browsing" I want to introduce a new method that nobody explored yet.
I talked of this idea on multiple occasions, here:
https://forums.linuxmint.com/viewtopic.php?p=2460391#p2460391
And here:
https://github.com/orgs/linuxmint/discussions/184
 in combination with tags.

Today I wanna talk again of my idea in the hope to reach a larger audience.

This new browsing system would **track most accessed folder paths** and save their name in a kind of "tag cloud", accessible from a panel in Nemo.  

**Most visited folders** would be highlighted in larger fonts and default home folders (pictures, documents, etc) could be highlighted in a single, calm colour to differentiate them from common folders

The benefit to store paths as "cloud tags" would be to grow a dynamically, constantly refined, user-specific "map" where you can **navigate to final destinations** and avoid digging manually into folder trees, because lets be serious, browsing is a time consuming, procrastination supporting, boring and unproductive chore to get to our files.
Finally only files count and folders add a complex organization. 

With this "modern browsing" system you'd learn a new and relaxing habit to get things done, instead to rely on manual digging. 
And this would be especially beneficial for deeply nested files you visit pretty often.

**Comment by Jopp-gh** [comment]
Here a quick mockup
![333147384-bfceeb4e-6ade-4aeb-a613-1280262d69f8](https://github.com/linuxmint/discussions/assets/48620536/fd6ae028-850d-4bff-9bba-2807c0b22bcc)

**Comment by GitHubIsToxic** [comment]
This very feature has in the past existed for years (from at least Mint 13 until Mint 17). It was erroneously removed 'upstream' by a confused GTK developer, and has been missing since Mint 18. This was through no fault of Linux Mint developers, mind you.

See here: https://github.com/orgs/linuxmint/discussions/1093
